pkg/engine/logging/default_logger.go

```diff
@@ -0,0 +1,43 @@
+package logging
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type defaultLogger struct {
+	entries []string
+}
+
+type logEventWrapper struct {
+	EventName string
+	Data      interface{}
+}
```

Comment:
I wonder for this and all the event payloads if we should add json struct tags and enforce consistent naming + style for fields (camel case?). Does not need to be done as part of this PR.